code/game/objects/items/holy_weapons.dm

```diff
@@ -386,31 +386,35 @@
 	icon = 'icons/obj/transforming_energy.dmi'
 	icon_state = "swordblue"
 	inhand_icon_state = "swordblue"
-	desc = "If you strike me down, I shall become more robust than you can possibly imagine."
+	worn_icon_state = "swordblue"
 	slot_flags = ITEM_SLOT_BELT
+	desc = "If you strike me down, I shall become more robust than you can possibly imagine."
 
 /obj/item/nullrod/claymore/saber/red
 	name = "dark energy sword"
 	icon_state = "swordred"
 	inhand_icon_state = "swordred"
+	worn_icon_state = "swordred"
 	desc = "Woefully ineffective when used on steep terrain."
 
 /obj/item/nullrod/claymore/saber/pirate
 	name = "nautical energy sword"
 	icon_state = "cutlass1"
 	inhand_icon_state = "cutlass1"
+	worn_icon_state = "swordred"
 	desc = "Convincing HR that your religion involved piracy was no mean feat."
 
 /obj/item/nullrod/sord
 	name = "\improper UNREAL SORD"
 	desc = "This thing is so unspeakably HOLY you are having a hard time even holding it."
```

Comment:
I assume that the lack of "that" between "HOLY" and "you" is a reference, yes?

Reply:
Yeah it's I believe a direct quote except SHITTY (or whatever term was used I forget) was replaced with HOLY. It parses like Hussie's writing at least.